pyproject.toml

```diff
@@ -0,0 +1,11 @@
+# Configuration for Black.
+
+# NOTE: you have to use single-quoted strings in TOML for regular expressions.
+# It's the equivalent of r-strings in Python.  Multiline strings are treated as
+# verbose regular expressions by Black.  Use [ ] to denote a significant space
+# character.
+
+[tool.black]
+line-length = 79
+target-version = ['py36', 'py37', 'py38']
+include = '\.pyi?$'
```

Comment:
newline?

Reply:
I tried this and then also to change the syntax in the GH Action to check `black` formatting and still set the same file called out.  Odd.